### DISCUSSION TITLE
Changing base maps

### DISCUSSION THREAD

**Discussion by technosprout** [Leaflet/Leaflet#9156] [Q&A]
I am looking at a series of base maps about 300 that I would like to scroll thru, is there a way to cache the tiles to make the feel more smooth and cinematic?
This is not for maps but large image sets think a 30 second 4K movie

**Comment by Falke-Design** [comment]
Normally the browser caches the tiles already. Else you create your own logic where you just set the opacity to 0 instead of removing the layer.
